ovn-tester/ovn_ext_cmd.py

```diff
@@ -1,7 +1,21 @@
 from collections import defaultdict
+from fnmatch import fnmatch
 from io import StringIO
 
 
+# The wrapper allows us to execute the command on all
+# matching central containers
+class CentralNodeWrapper(object):
```

Comment:
I think a better way to define this class would be as follows:
```
class CentralNodeWrapper(ovn_sandbox.Sandbox):

```
And that's it. This way, you don't have to override the `run()` method, and you get a much more thorough `run()` method this way as well. I think having a wrapper class is still a good idea in case more specialization is needed within `ovn_ext_cmd.py`.

Reply:
Good idea, done.